.github/workflows/submit.yml

```diff
@@ -298,7 +298,6 @@ jobs:
           java -version
           which ant
           ant -version
-          sudo xcode-select --switch /Applications/Xcode_14.3.1.app/Contents/Developer
```

Comment:
do you think it might be better to comment out this line, in case we need emergency override?

Reply:
Sure. It might also save a question from someone wondering why we do an `xcode-select` for macOS 13 but not macOS 14